cgmes/cgmes-model/src/test/java/com/powsybl/cgmes/model/CgmesOnDataSourceTest.java

```diff
@@ -58,26 +59,127 @@ void testExists(String testName, String filename, String cimVersion, boolean exp
         assertEquals(expectedExists, exists);
     }
 
-    @Test
-    void testFileDoesNotExist() throws IOException {
+    static Stream<Arguments> provideArgumentsFortestXmlMainFileXiidmZip() {
+        return Stream.of(
+                Arguments.of("foo", "xml"),
+                Arguments.of("foo", null),
+                Arguments.of("foo", ""),
+                Arguments.of("foo", "notexists"),
+                Arguments.of("bar", "xml"),
+                Arguments.of("bar", null),
+                Arguments.of("bar", ""),
+                Arguments.of("bar", "notexists")
```

Comment:
are they useful? I think each one corresponds to the same logic as one of the previous arguments.